js-miniapp-sdk/src/event-types/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Enum for supported SDK event types
+ */
+export enum MiniAppEvents {
+  externalwebviewclose = 'miniappwebviewclosed',
+  pause = 'miniapppause',
+  resume = 'miniappresume',
```

Comment:
Maybe the enum names should use all caps with snake case? It's the format we used for most other enums in the SDK:
```suggestion
  EXTERNAL_WEBVIEW_CLOSE = 'miniappwebviewclosed',
  PAUSE = 'miniapppause',
  RESUME = 'miniappresume',
```